controllers/user-controller.js

```diff
@@ -0,0 +1,50 @@
+const userService = require('../service/user-service');
+const {validationResult} = require('express-validator');
+const ApiError = require('../api-error');
+
+class UserController {
+    async register(req, res, next) {
+        try {
+            const errors = validationResult(req);
+
+            if(!errors.isEmpty()){
+                return next(ApiError.BadRequest("Ошибка при валидации", errors.array()))
+            }
+
+            const {email, password} = req.body;
+            const userData = await userService.register(email, password);
+
+            res.cookie('refreshToken', userData.refreshToken, {maxAge: 30 * 24 * 60 * 60 * 1000, httpOnly: true});
```

Comment:
It's not very obvious how big the max age is. You can create a separate constant (like `const THIRTY_DAYS = ...;`) in some `constants.js` file and export it using here. Also, it's reused later - code duplicating is a bad practice.